lib/scan_result.dart

```diff
@@ -0,0 +1,70 @@
+part of flutter_ble_lib;
+
+abstract class _Metadata {
+  static const String ID = "id";
```

Comment:
https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names

Reply:
For the sake of conciseness I renamed all const fields' names to match lowerCamelCase.